documentation/oracle/governance/UMIPs.md

```diff
@@ -0,0 +1,69 @@
+# UMIPs
+
+UMIPs (“UMA Improvement Proposals”) are the design documents used to propose changes to the UMA ecosystem. 
+They provide information to the UMA community that describes a new feature for the UMA protocol, or its ecosystem. 
+The UMIP should provide a concise technical specification of the feature and a rationale for the feature. 
+They are modeled after [EIPs](https://eips.ethereum.org/) and [ZEIPs](https://blog.0xproject.com/0x-protocol-governance-voting-walkthrough-and-faq-3becfd57a370). 
+See here for an [EIP template](https://github.com/ethereum/EIPs/blob/master/eip-template.md) and [ZEIP template](https://github.com/0xProject/ZEIPs/blob/master/ISSUE_TEMPLATE.md). 
+
+We intend UMIPs to be the primary mechanism for proposing new features, collecting community technical input on an issue, and for documenting the design decisions that have gone into the UMA protocol.
+UMIPs are a convenient way to track the progress of an implementation. 
+
+# What is the lifecycle of a UMIP? 
+
+A successful UMIP will move along the following stages: Draft ⟶ Last Call ⟶ Approved ⟶ Final. 
+Unsuccessful states are also possible: Abandoned and Rejected.
+
+## Draft
+A UMIP that is open for consideration and is undergoing rapid iteration and changes. 
+In order to proceed to “Last Call,” the implementation must be complete. 
+Every UMIP author is responsible for facilitating conversations and building community consensus for the proposal.
+
+## Last Call
+A UMIP that is done with its initial iteration and ready for review by a wide audience. 
+If upon review, there is a material change or substantial unaddressed complaints, the UMIP will revert to "Draft". 
+If the UMIP requires code changes, the core devs must approve the UMIP and the changes must be merged into the protocol repository. 
+A successful UMIP will be in "Last Call" status for a reasonable period of time for comments and be merged (if necessary) before moving to a tokenholder vote. 
+
+## Approved
+A UMIP that successfully passes the "Last Call" will be put to UMA tokenholder vote. 
+
+## Final
+If tokenholders approve the proposal, the live protocol will be updated to reflect them. The UMIP is then considered to be in the "Final" state. 
+
+## Abandoned
+If at any point during the UMIP Finalization Process, a UMIP is abandoned, it will be labeled as such. 
+Grounds for abandonment include a lack of interest by the original author(s), or it may not be a preferred option anymore.
+
+## Rejected
```

Comment:
Does this also include proposals that were rejected by the token holders?

Reply:
Good question - my guess is yes.

Reply:
I don't think we've considered that, so yes, let's make it include that

Reply:
This is clarified in a previous commit, where I added:
If tokenholders do not approve a proposal, or the UMIP is fundamentally broken or rejected by the core team, it will not be implemented.